.env.example

```diff
@@ -6,20 +6,4 @@ NEXT_PUBLIC_BACKEND_URL=
 NEXT_PUBLIC_MAINNETFORK_SUPPORTED=true
 NEXT_PUBLIC_ARBITRUMFORK_SUPPORTED=true
 NEXT_PUBLIC_ARBITRUM_SUPPORTED=true
-NEXT_PUBLIC_CURVE_GRAPH_URL=https://api.thegraph.com/subgraphs/name/convex-community/curve-pools
```

Comment:
I respect the intention with this but these really don't need to be env vars. It's just difficult to keep track of and clunky to manage on vercel.